packages/date-picker/src/DatePicker/DatePicker.tsx

```diff
@@ -80,7 +81,7 @@ export enum DayOfWeek {
 
 export const DatePicker: React.FunctionComponent<DatePickerProps> = ({
```

Comment:
As per the recent discussions about using VFC instead of FC

```suggestion
export const DatePicker: React.VFC<DatePickerProps> = ({
```